lib/media/streaming_engine.js

```diff
@@ -1588,15 +1591,36 @@ shaka.media.StreamingEngine = class {
 
     const hasClosedCaptions = stream.closedCaptions &&
         stream.closedCaptions.size > 0;
-    if ((stream.emsgSchemeIdUris != null &&
-        stream.emsgSchemeIdUris.length > 0) ||
-        this.config_.dispatchAllEmsgBoxes) {
-      new shaka.util.Mp4Parser()
+
+    let parser;
+    const hasEmesg = ((stream.emsgSchemeIdUris != null &&
+      stream.emsgSchemeIdUris.length > 0) ||
+      this.config_.dispatchAllEmsgBoxes);
+    const shouldParsePrftBox =
+      (this.config_.parsePrftBox && !this.parsedPrftEventRaised_);
```

Comment:
You only ever raise the event once per playback session?  Why?  Should this perhaps be documented in the config?

Reply:
The choice to only raise this event once was motivated by performance, as we would only need to know the start date once, as it would be the same across all bitrates.

Reply:
Could you please document this in the jsdoc for the parsePrftBox config?

Reply:
I have updated the config jsdoc.